bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/SiderealDetailEditor.scala

```diff
@@ -0,0 +1,130 @@
+package jsky.app.ot.gemini.editor.targetComponent.details
+
+import java.awt.{ Insets, GridBagConstraints, GridBagLayout }
+import javax.swing.{ JLabel, JPanel, JComponent }
+
+import edu.gemini.pot.sp.ISPNode
+import edu.gemini.shared.util.immutable.{ Option => GOption }
+import edu.gemini.spModel.obs.context.ObsContext
+import edu.gemini.spModel.target.SPTarget
+import edu.gemini.spModel.target.system.{HmsDegTarget, ITarget}
+import jsky.app.ot.gemini.editor.targetComponent.{ GuidingFeedbackEditor, MagnitudeEditor}
+
+import scalaz.syntax.id._
+
+final class SiderealDetailEditor extends TargetDetailEditor(ITarget.Tag.SIDEREAL) {
+  import NumericPropertySheet.Prop
+
+  // Editor Components
+
+  val kind   = new TargetTypeEditor
+  val name   = new SiderealNameEditor
+  val coords = new CoordinateEditor
+
+  val mags   = new MagnitudeEditor <| { e =>
+    e.getComponent.asInstanceOf[JComponent].setBorder(titleBorder("Magnitudes"))
+  }
+
+  val props = NumericPropertySheet[HmsDegTarget]("Motion", _.getTarget.asInstanceOf[HmsDegTarget],
+    Prop("∆ RA",     "mas/year", _.getPM1),
```

Comment:
I think the `delta` deserves some checks working on Windows and Linux, they may not look ok